build/module-builder/package.json

```diff
@@ -39,6 +39,7 @@
     "style-loader": "^0.23.0",
     "tar": "^4.4.6",
     "ts-loader": "^6.0.0",
+    "typescript": "^3.9.7",
```

Comment:
This should not be required, it will use typescript provided by the root node_modules

Reply:
The reason why I specify the typescript version here is that otherwise, the module builder would use the version installed by `typescript-json-schema`.

```bash
build/module-builder |> yarn why typescript
yarn why v1.22.10
[1/4] Why do we have the module "typescript"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "typescript@3.7.5"
info Reasons this module exists
   - "typescript-json-schema" depends on it
   - Hoisted from "typescript-json-schema#typescript"
info Disk size without dependencies: "48.9MB"
info Disk size with unique dependencies: "48.9MB"
info Disk size with transitive dependencies: "48.9MB"
info Number of shared dependencies: 0
Done in 0.30s.
```

Since it installs typescript version 3.7.5, when I try to build the google-speech module, I have those typing error:

```bash
[module-builder] [ERROR] in /home/laurentlp/code/botpress/botpress/modules/google-speech/node_modules/google-gax/build/src/index.d.ts (47,10)
                 TS1141: String literal expected.
                 
[module-builder] [ERROR] in /home/laurentlp/code/botpress/botpress/modules/google-speech/node_modules/google-gax/build/src/index.d.ts (47,13)
                 TS1005: ';' expected.
                 
[module-builder] [ERROR] in /home/laurentlp/code/botpress/botpress/modules/google-speech/node_modules/google-gax/build/src/index.d.ts (47,13)
                 TS2304: Cannot find name 'protobufMinimal'.
                 
[module-builder] [ERROR] in /home/laurentlp/code/botpress/botpress/modules/google-speech/node_modules/google-gax/build/src/index.d.ts (47,29)
                 TS1005: ';' expected.
                 
[module-builder] [ERROR] in /home/laurentlp/code/botpress/botpress/modules/google-speech/node_modules/google-gax/build/src/index.d.ts (47,29)
                 TS2304: Cannot find name 'from'.
                 
[module-builder] [ERROR] in /home/laurentlp/code/botpress/botpress/modules/google-speech/node_modules/google-gax/build/src/index.d.ts (47,34)
                 TS1005: ';' expected.
```

Reply:
Can you upgrade typescript-json-schema instead ? There's a ton of dependencies which needs to be upgraded, but since we have to do a minimal test when doing so, every little bit helps

Reply:
Yes good idea, but still, this means that we would depend on the version of typescript that `typescript-json-schema` installs which I think could be error prone. Personally, I would do both. Make sure packages are up-to-date by bumping their version little by little and explicitly specifying which version of typescript we require for this package. What do you think?